javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -380,7 +380,7 @@ function normalizeProxyConfiguration(config) {
     }
   } else if ('pac' === config.proxyType) {
     if (config.proxyAutoconfigUrl && !config.pacUrl) {
-      config.pacUrl = config.proxyAutoconfigUrl;
+      config.proxyAutoconfigUrl = config.proxyAutoconfigUrl;
```

Comment:
Actually, it looks like the entire `normalizeProxyConfiguration` function isn't needed anymore